sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernatePaginatedDataSource.java

```diff
@@ -176,63 +176,63 @@ public default void restrictPaginationByDate(Criteria criteria, Date start, Date
     if (property != null) {
       criteria.add(Restrictions.between(property, start, end));
     } else {
-      errorHandler.accept(getFriendlyName() + " has no " + type.name().toLowerCase() + " date.");
+      errorHandler.accept(String.format("%s has no %s date.", getFriendlyName(), type.name().toLowerCase()));
     }
   }
 
   @Override
   default void restrictPaginationByExternalName(Criteria criteria, String name, Consumer<String> errorHandler) {
-    errorHandler.accept(getFriendlyName() + " has no external name.");
+    errorHandler.accept(String.format("%s has no external name.", getFriendlyName()));
   }
 
   @Override
   default void restrictPaginationByFulfilled(Criteria criteria, boolean isFulfilled, Consumer<String> errorHandler) {
-    errorHandler.accept(getFriendlyName() + " has no fulfillment (nor existential dread).");
+    errorHandler.accept(String.format("%s has no fulfillment (nor existential dread).", getFriendlyName()));
   }
 
   @Override
   public default void restrictPaginationByHealth(Criteria criteria, EnumSet<HealthType> healths, Consumer<String> errorHandler) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "healths" object should be an interface such as "Set" rather than the implementation "EnumSet". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)